app/src/androidTest/java/com/codesses/priceconverter/ExampleInstrumentedTest.kt

```diff
@@ -1,11 +1,13 @@
-package com.minimalisticapps.priceconverter
+package com.codesses.priceconverter
```

Comment:
why this rename? it should be kept as `minimalisticapps`

Reply:
It's updated